src/timeline/js/controllers.js

```diff
@@ -1088,8 +1088,8 @@ $scope.SetTrackLabel = function (label){
  	return transition_size;
  };
  
- // Search through clips and transitions to find the closest element within a given threashold
- $scope.GetNearbyPosition = function(pixel_positions, threashold, ignore_ids){
+ // Search through clips and transitions to find the closest element within a given threshold
+ $scope.GetNearbyPosition = function(pixel_positions, threshold, ignore_ids){
```

Comment:
source change, please review

Reply:
+1, this gets both occurrences of the "threashold" variable, which is only used at these two points in the code.